frontend/src/graphql/queries/PendingEdits.gql

```diff
@@ -0,0 +1,8 @@
+query PendingEdits($type: TargetTypeEnum!, $id: ID!) {
```

Comment:
Maybe should be named `PendingEditsCount`, or some other thing?

Reply:
Yeah, `PendingEditsCount` sounds fine to me.